tutorials/model_command.md

```diff
@@ -0,0 +1,184 @@
+\page model_command Model Command
+
+## Overview
+`ign model` command allows you to get information about the models for a given running `ignition` simulation.
+
+For each model, it is possible to get information about its
+ -  Pose: Pose of the model
+ -  Links: Pose, mass, and inertial matrix of the link
+ -  Joints: Parent link, child link and joint type.
+
+### Available options for the model command
+
+```
+  --list                     Get a list of the available models.        
+  -m [--model] arg           Select the model to be shown.              
+  -l [--link]  arg           Select a link to show its properties.      
+                             If no arg is passed all links are printed  
+                             Requires the -m option                     
+                                                                        
+  -j [--joint] arg           Select a joint to show its properties.     
+                             If no arg is passed all joints are printed 
+                             Requires the -m option                     
+
+  -h [ --help ]              Print the help message.
+```
```

Comment:
This is not matching the current output. In fact, it will probably be difficult to keep these in sync.

Reply:
Addressed
It was not supposed to be matching the --help option, but to be a short list of the options.
I ended up removing the command list now as it can be listed with the --help and as you said, the tutorial one might get outdated at some point.